src/components/atoms/ImageInput/ImageInput.scss

```diff
@@ -0,0 +1,70 @@
+@import "scss/constants.scss";
+
+.ImageInput {
+  &__container {
+    position: relative;
+    display: flex;
+    padding: 2.5em 0;
+    align-items: center;
+    justify-content: center;
+    border-radius: 22px;
```

Comment:
Use `constants.scss` in the whole file.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1988/commits/c96f9dd32da5df5490a68899f63f7f59ea7943ab